nuxt/pages/frise/_procedureId/index.vue

```diff
@@ -95,7 +95,7 @@
               </v-list-item-title>
             </v-list-item> -->
 
-            <v-dialog v-model="dialog" width="500">
+            <v-dialog v-model="dialog" v-if="$user.canDeleteProcedure()" width="500">
```

Comment:
J'ai un nouveau warning du linter 
```
warning  Attribute "v-if" should go before "v-model"  vue/attributes-order`
```

```suggestion
            <v-dialog v-if="$user.canDeleteProcedure()" v-model="dialog" width="500">
```